package.json

```diff
@@ -86,6 +86,7 @@
     "lint-staged": "12.4.0",
     "pre-commit": "1.2.2",
     "prettier-standard": "16.4.1",
+    "shx": "^0.3.4",
```

Comment:
Thanks for the contrib, but I will not approve this PR with this package.  `cp` and `rm` is a pretty common command across unix distros and the flags used should be common as well.

Reply:
Those commands don't exist in windows, so I added the `shx` to make the build cross platform.  

I'll remove the package from the PR and just have it in my local to be able to run them.  
In this case I'd recommend adding this as an advice for developers working in windows.